inc/metadata/namespace.php

```diff
@@ -975,7 +975,7 @@ function add_json_ld_metadata() {
 	if ( $context === 'section' ) {
 		global $post;
 		$section_information = get_section_information( $post->ID );
-		$book_information = Book::getBookInformation( null, false );
+		$book_information = Book::getBookInformation( null, false, false );
```

Comment:
Adding additional parameter where we want to make sure we don't want to read contributors as a string stored in the cache if some other method or action did it. It will force to read the contributors from the database instead.